spec/fixtures/transactions.yml

```diff
@@ -6,7 +6,7 @@ one:
   updated_at: January 1, 2012
 
 two:
-  invoice_id: 980190962
+  invoice_id: 1
```

Comment:
i changed this so that there was only one record associated with the invoice. this isn't a real id...we can talk about how we want to set up relationships going forward